code/modules/clothing/masks/gasmask.dm

```diff
@@ -65,6 +65,22 @@
 	icon_state = "clown"
 	item_state = "clown_hat"
 
+obj/item/clothing/mask/gas/clown_hat/attack_self(mob/user)
+
+	var/mob/M = usr
+	var/list/options = list()
+	options["True Form"] = "clown"
+	options["The Feminist"] = "sexyclown"
+	options["The Madman"] = "joker"
+	options["The Rainbow Color"] ="rainbow"
+
+	var/choice = input(M,"To what form do you wish to Morph this mask?","Morph Mask") in options
```

Comment:
```suggestion
	var/choice = input(M,"To what form do you wish to Morph this mask?","Morph Mask") as anything|nul in options
```
This will add `cancel` button.